examples/motoko_examples/whoami/test/test.ts

```diff
@@ -0,0 +1,104 @@
+import {
+    run_tests,
+    Test
+} from 'azle/test';
+import { execSync } from 'child_process';
+import { createActor } from './dfx_generated/whoami';
+import { Ed25519KeyIdentity } from '@dfinity/identity';
+import { SignIdentity } from '@dfinity/agent';
+
+function createIdentity(seed: number): SignIdentity {
+    const seed1 = new Array(32).fill(0);
```

Comment:
Please explain this to me

Reply:
We talked about this offline. I was just following the precedence set by @tarek-eg. We explored a different way to do this without mutating so I will make some slight changes, but the rough idea is to just generate a random seed for the identity function.